### ISSUE TITLE
font-weight no corresponde

### ISSUE THREAD

**Issue by dennisamstein** [coma/msds#3]
Hello,

Estaba creando los estilos con la tipografía "Core Sans C" (9 weights)
En Zeplin el peso ExtraBold me daba 800, pero luego al importarlo en sketch me sale el Heavy.
El peso realmente se llama Extrabold85 (8 es weight y 5 es el wide). Intenté con 700 y me pone el Bold (casi). Luego intenté poner 750 y en la visualización del dashboard me parece que es el correcto, pero al importar el proyecto a sketch me da error de invalid project.
adjunto unos screens

Tiene muy buena pinta el generador de DS,
saludos!
![fontweight](https://user-images.githubusercontent.com/44170940/46962066-6efaa700-d0a2-11e8-8245-c84f8850280f.jpg)

**Comment by eduardo-syltek** [comment]
Sí, el tema de mapear correctamente fuentes de sistema y de web me tiene un poco amargado, estamos pensando como darle la mejor solución posible.

**Comment by coma** [comment]
@dennisamstein tenemos una solución bastante óptima para esto en camino, dejo abierto hasta que la suba a prod.
